handbook/company/communications.md

```diff
@@ -164,6 +164,11 @@ Recurring expenses related to a particular team member, such as coworking fees,
 - Individualized expenses are always attributed to the  "🔦 Business operations" department.
 - These expenses are still considered non-personnel expenses, in the same way seat licenses for tools like Salesforce or GitHub are considered non-personnel expenses.   
 
+### Expense acquittal process
```

Comment:
- I think this would make sense as an h4 under `### Brex`, not an h3 alongside it.  Better yet, I think we could consolidate this under "Brex" as a bullet point.
- I've never heard of the term "acquittal" so I looked it up: https://staffhelp.secure.griffith.edu.au/app/answers/detail/a_id/3845/~/what-is-an-acquittal%3F-  I think we could avoid using this term for it, and ask that people put clear memos on expenses.
- While not having receipts makes it harder, clear memos would solve the problem in terms of expense categorization. 
- Love the prefix idea for memos and attributing expenses.  Sounds like we would need more steps before this could be used for reporting rollups though (e.g. a reference of event prefixes)

Reply:
Joanne: If it's just memos and not receipts, then I _think_ it would be fine.  The receipts are helpful when we need travel needing to be changed.